operator/v1/types_console.go

```diff
@@ -22,12 +22,26 @@ type ConsoleSpec struct {
 	// customization options to the web console.
 	// +optional
 	Customization ConsoleCustomization `json:"customization"`
+	// provides contains configuration for using a specific service provider.
+	Providers ConsoleProviders `json:"providers"`
 }
 
 type ConsoleStatus struct {
 	OperatorStatus `json:",inline"`
 }
 
+type ConsoleProviders struct {
+	// statusPage contains ID for statuspage.io page that provides status info about.
+	// +optional
+	StatusPage *StatusPageProvider `json:"statusPageIO,omitempty"`
+}
+
+// StatusPageProvider provides identity for statuspage account.
+type StatusPageProvider struct {
+	// id is the statuspage ID.
+	ID string `json:"id"`
```

Comment:
```suggestion
	PageID string `json:"pageID"`
```

Reply:
PageID?

Reply:
Yes, it's specifically a page-id in the Statuspage API.